Error: Cannot find module 'uuid/v4'

**Describe the bug**
Error:
![image](https://user-images.githubusercontent.com/46999684/102302606-0baa7f00-3f62-11eb-9057-e6d21b16f39f.png)

**System**

Browser version: Chrome 87.0.4280.88
Node.js version: v10.23.0
NPM or Yarn version: npm 6.14.9
Operating system:

**Actual behavior**

[0] [nodemon] 2.0.6
[0] [nodemon] to restart at any time, enter `rs`
[0] [nodemon] watching path(s): *.*
[0] [nodemon] watching extensions: js,mjs,json
[0] [nodemon] starting `node index.js`
[0] /root/network/api/utils/cloudinary.js:1
[0] Error: Cannot find module 'uuid/v4'
[0] Require stack:
[0] - /root/network/api/utils/cloudinary.js
[0] - /root/network/api/resolvers/user.js
[0] - /root/network/api/resolvers/index.js
[0] - /root/network/api/server.js
[0] - /root/network/api/index.js
[0]     at new MODULE_NOT_FOUND (/root/my-network/api/node_modules/esm/esm.js:1)
[0]     at wu (/root/my-network/api/node_modules/esm/esm.js:1)
[0]     at Eu (/root/my-network/api/node_modules/esm/esm.js:1)
[0]     at ku (/root/my-network/api/node_modules/esm/esm.js:1)
[0]     at Proxy.<anonymous> (/root/my-network/api/node_modules/esm/esm.js:1)
[0]     at Object.<anonymous> (/root/my-network/api/node_modules/esm/esm.js:1)
[0]     at Tu (/root/my-network/api/node_modules/esm/esm.js:1)
[0]     at Mu (/root/my-network/api/node_modules/esm/esm.js:1)
[0]     at Object.import (/root/my-network/api/node_modules/esm/esm.js:1)
[0]     at Object.<anonymous> (/root/my-network/api/utils/cloudinary.js:1)
[0] [nodemon] app crashed - waiting for file changes before starting...
[1] ℹ ｢wds｣: Project is running at http://192.168.1.125/
[1] ℹ ｢wds｣: webpack output is served from
[1] ℹ ｢wds｣: Content not from webpack is served from /root/network/frontend/public
[1] ℹ ｢wds｣: 404s will fallback to /
[1] Starting the development server...
[1]
[1] Compiled successfully!

--- a/api/utils/cloudinary.js
+++ b/api/utils/cloudinary.js
@@ -1,5 +1,5 @@
 import cloudinary from 'cloudinary';
-import uuid from 'uuid/v4';
+import { v4 as uuid } from 'uuid';
 
 cloudinary.config({
   cloud_name: process.env.CLOUDINARY_CLOUD_NAME,

